walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,12 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    String n ="";
```

Comment:
In general it might be helpful to you and others reading your code to have a more meaningful variable name. Maybe you could call it greetingName or nameToGreet or something like that?